### PR TITLE
Add backend logging

### PR DESCRIPTION
**Pull request by anetteu**
Solves #58

### CODE REVIEW

**Comment by anetteu** [comment]
This PR implements logging of api exceptions. Logging api requests and responses is left for #84

File: backend/src/flotilla/api/missions_api.py [hunk 1]
```diff
@@ -73,6 +79,7 @@ async def read_single_mission(
     try:
         mission: Mission = mission_deserializer(echo_response.json())
     except Exception:
+        logger.exception(f"Failed to get mission with id {mission_id}.")
```
Comment (vetlek) [inline backend/src/flotilla/api/missions_api.py:82]: Log the exception

File: backend/src/flotilla/api/missions_api.py [hunk 2]
```diff
@@ -33,6 +36,7 @@ async def read_missions(
     try:
         echo_response: RequestResponse = echo_requests.get_missions()
     except RequestException:
+        logger.exception("Could not get missions from echo.")
```
Comment (vetlek) [inline backend/src/flotilla/api/missions_api.py:39]: Log the exception

**Review by vetlek — CHANGES_REQUESTED** [review]
Some minor things

File: backend/src/flotilla/api/missions_api.py [hunk 3]
```diff
@@ -65,6 +70,7 @@ async def read_single_mission(
     try:
         echo_response: RequestResponse = echo_requests.get_mission(mission_id)
     except RequestException:
+        logger.exception(f"Failed to get mission with id {mission_id}.")
```
[the remark this answers is not in the document]
Comment (anetteu) [reply backend/src/flotilla/api/missions_api.py:73]: The exception is logged when using `logger.exception()`, so this is already handled

**Review by vetlek — APPROVED** [review]

**Pull request merged**